ultron/src/main/java/com/atiurin/ultron/extensions/ViewInteractionExt.kt

```diff
@@ -20,6 +24,17 @@ fun ViewInteraction.withAssertion(assertion: OperationAssertion) = UltronEspress
 fun ViewInteraction.withAssertion(name: String = "", isListened: Boolean = false, block: () -> Unit) =
     UltronEspressoInteraction(this).withAssertion(DefaultOperationAssertion(name, block.setListenersState(isListened)))
 
+//root view searching
```

Comment:
Pls, add a javadoc with description of the method purpose. User should understand the usecase

Reply:
https://github.com/open-tool/ultron/wiki/Avoiding-nontrivial-element-lookup-exceptions